versioned_docs/version-6.x.x/getting-started.md

```diff
@@ -49,11 +49,17 @@ const sequelize = new Sequelize('database', 'username', 'password', {
 });
 ```
 
-The Sequelize constructor accepts a lot of options. They are documented in the [API Reference](pathname:///api/v6/class/src/sequelize.js~Sequelize.html#instance-constructor-constructor).
+The Sequelize constructor accepts many options which are documented in the [API Reference](pathname:///api/v6/class/src/sequelize.js~Sequelize.html#instance-constructor-constructor).
+
+:::note
+
+In the example above, `Sequelize` refers to the library itself while `sequelize` refers to an instance of Sequelize, which represents a connection to one database. This convention will be followed throughout the documentation.
+
+:::
 
 ### Testing the connection
 
-You can use the `.authenticate()` function to test if the connection is OK:
+To test if the connection is OK, use the `.authenticate()` function:
```

Comment:
For this specific sentence, I prefer the old version because the use of "can" helps indicate that this is completely optional

Reply:
```suggestion
To test if the connection is OK, you can use the `.authenticate()` function:
```

Reply:
Yep, perfect :)